OrbitCore/SymbolHelper.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
```

Comment:
Is this a new file? I think we agreed at some point to not use #pragma once and prefer #ifndef <HEADER> #define <HEADER>.

Reply:
Okay, I wasn't aware -> done